app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        person_list.append(Person(person["name"], person["age"]))
+    for i in range(len(people)):
+        if people[i].get("wife") and people[i]["wife"] is not None:
```

Comment:
This check is tooo overcomplicated. I bet you can quickly come up with an easier solution if you only remember what returns `get()` method if there is no such key in dict